bindings/python/scripts/utils.py

```diff
@@ -93,17 +95,22 @@ def parse_arguments(script):
         )
         parser.add_argument(
             "--json_output_path",
-            default=get_parent_directory(file=__file__),
+            default=os.getcwd(),
```

Comment:
This will produce different outputs depending on the execution path of the python file.

Reply:
The existing solution would create it in the installed folders.